packages/cli/src/lib/push/addCommon.ts

```diff
@@ -0,0 +1,85 @@
+import {CliUx} from '@oclif/core';
+import {red, dim, green} from 'chalk';
+import {BuiltInTransformers, errors} from '@coveo/push-api-client';
+import {normalizeInvalidFields} from '../flags/sourceCommonFlags';
+import {pluralizeIfNeeded} from '../utils/string';
+import dedent from 'ts-dedent';
+
+const allowedCharRegExp = new RegExp('^[a-z]+[a-z0-9_]*$');
+
+export const handleAddError = (err: unknown) => {
+  const {UnsupportedFieldError} = errors;
+  if (err instanceof UnsupportedFieldError) {
+    CliUx.ux.error('Invalid field name detected while parsing your data.', {
+      exit: false,
+    });
+    const normalizations = err.unsupportedFields.map((f) => {
```

Comment:
avoid using single letter for your callback.
Yes, it might be less verbose, but is also less clear at first glance.